c2_components/src/mfx_c2_encoder_component.cpp

```diff
@@ -1801,7 +1879,7 @@ void MfxC2EncoderComponent::DoUpdateMfxParam(const std::vector<C2Param*> &params
                 mfxStatus sts = MFX_ERR_NONE;
                 switch (m_bitrateMode->value) {
                     case C2Config::BITRATE_CONST:
-                        sts = mfx_set_RateControlMethod(MFX_RATECONTROL_CBR, &m_mfxVideoParamsConfig);
+                        sts = mfx_set_RateControlMethod(MFX_RATECONTROL_CQP, &m_mfxVideoParamsConfig);
```

Comment:
did this pass the CTS?
bitrate_const corresponds to CBR is defined by here, please don't modify it directly.
http://inlubt0232.iind.intel.com:8080/source/xref/CELADON/frameworks/av/media/codec2/sfplugin/utils/Codec2Mapper.cpp?r=e9707b37#96

Reply:
removed